crates/db_views/src/post_view.rs

```diff
@@ -1988,4 +1991,54 @@ mod tests {
 
     cleanup(data, pool).await
   }
+
+  #[tokio::test]
+  #[serial]
+  async fn speed_check() -> LemmyResult<()> {
+    let pool = &build_db_pool().await?;
+    let pool = &mut pool.into();
+    let data = init_data(pool).await?;
+
+    // Make sure the post_view query is less than this time
+    let duration_max = Duration::from_millis(40);
```

Comment:
Based on the results of my local machine of most of these being ~30ms for 1k posts, I've set the fail check at 40ms. This means this PR will fail in production until we find the culprit.